hack/start-api

```diff
@@ -27,6 +27,17 @@ apiserver_flags="
 ld_flags="$(./hack/get-build-ld-flags)"
 
 case $(k8s_env) in
+    $KIND)
+        echo "Found kind ..."
+        kubectl -n garden port-forward service/etcd 32379:2379 > /dev/null &
```

Comment:
Can't this be avoided by talking from your machine to the etcd running in kind? Or is that not possible?